lib/Controller/MessagesController.php

```diff
@@ -671,7 +671,7 @@ public function setTag(int $id, string $imapLabel): JSONResponse {
 
 		try {
 			$tag = $this->mailManager->getTagByImapLabel($imapLabel, $this->currentUserId);
-		} catch (DoesNotExistException $e) {
+		} catch (ClientException $e) {
```

Comment:
Alternative: Not catching the `ClientException` and let `ErrorMiddleware` handle it. I needed to change the code because the wrong exception was documented.